runners/direct-java/pom.xml

```diff
@@ -77,95 +163,6 @@
         </executions>
       </plugin>
 
-      <!-- Ensure that the Maven jar plugin runs before the Maven
-        shade plugin by listing the plugin higher within the file. -->
-      <plugin>
```

Comment:
This caused surefire and jar to switch orders so jar comes first. I don't really know what the implications are here. Since we are using surefire for both unit tests and integration tests it is really about the individual executions, yes?